jbpm-human-task/jbpm-human-task-workitems/src/test/java/org/jbpm/services/task/wih/util/HumanTaskHandlerHelperTest.java

```diff
@@ -63,7 +65,86 @@ public void testSetDeadlinesNotStartedReassign() {
 		long expirationTime = deadlines.getStartDeadlines().get(0).getDate().getTime() - System.currentTimeMillis();
 		
 		assertEquals(4, roundExpirationTime(expirationTime));
-		
+	}
+
+	@Test
```

Comment:
I think we are missing test cases for:
R/<startDate>/<duration>
R/<duration>/<endDate>
R/<startDate>/<endDate>

I have only found dates and durations separately.

Reply:
added